src/api/FactsController.js

```diff
@@ -0,0 +1,26 @@
+export async function retrieveAnimalFact() {
+  const animalTypes = ['dog', 'cat'];
+  const randomNumber = randomIntFromInterval(0, animalTypes.length - 1);
+  const randomAnimalType = animalTypes[randomNumber];
+  const FactUrls = {
+    SENTENCE_URL: `https://cat-fact.herokuapp.com/facts/random?animal_type=${randomAnimalType}&amount=1`,
+    IMG_URL: `https://api.the${randomAnimalType}api.com/v1/images/search?size=full`,
+  };
```

Comment:
if you want to keep it in this convention, please move it outside the function, to the first lines of the file